### PR TITLE
Why estimate

### PR DESCRIPTION
**Pull request by HanCork89**
Note - we still need to add our gravatar account & image
### Blog title
Why Estimate
### Blog description
Is estimating ultimately a waste of time?
#### Author
- [X] I have checked for grammatical errors
- [X] I have checked for spelling errors
- [X] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by carolinewoodhams** [comment]
Taking (1)

File: _posts/2016-08-26-why-estimate [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+excerpt: Is estimating ultimately a waste of time?
+---
+
+
+##What we did##
+For some reason, we thought it would be a good idea to open up a debate in our Web Team around “What’s the point in estimating?”. It’s usually a hot topic in agile circles (#NoEstimates). We expected some strong points of view and a lively exchange of opinions. What we actually got was a good natured, rather polite room full of Engineers, Architects, Testers, Scrum Masters and Pod Leads, all making well constructed arguments for and against estimating.
+
+We ran the session not with a hope of finding a definitive answer, but more to see if we had general agreement around why we might or might not estimate. Since working with Scrum, estimating tickets is something the Web Team at Holiday Extras has always done; and even before that, in one form or another, when working with Kanban or on projects.
+
+So, do we actually need to estimate? When is the best time to estimate? Who benefits from estimation? Each pod uses a variation on estimating, but did that matter? Here’s a summary of a few of the arguments for and against…...
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: Don't use ellipses at the end of the sentence, makes it seem like you're uncertain of yourselves. Command the attention of the reader and end with a full stop.

File: _posts/2016-08-26-why-estimate [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+##In the red corner, arguments for...##
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: Again, don't use ellipses - a colon would do here.

File: _posts/2016-08-26-why-estimate [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Each pod uses a different method, but that doesn’t matter; estimation creates and encourages conversation about a piece of work and ensures the whole team understands completely the approach and any issues they’d face in taking on a ticket. This is particularly helpful for new starters as the detailed discussion increases their understanding of the nature of the work and how the established team view that work. Estimates may have to increase for a new starter whilst they find their feet, but that will be encompassed in the value that ticket is given and so is very easily scalable.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: 'ensures the whole team understands completely' - don't split infinitives, should read 'completely understands'

Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: I'm not clear what the last sentence in this paragraph is trying to convey:
"Estimates may have to increase for a new starter whilst they find their feet, but that will be encompassed in the value that ticket is given and so is very easily scalable."
Could you maybe reword this, even break it down into two sentences, as the point here isn't an immediate takeaway.

File: _posts/2016-08-26-why-estimate [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+If the discussion of the ticket is what’s important for the team, then the argument could be that we stop at the end of that discussion. However, we’ve found that putting a value on it - be that a Fibonacci, T-Shirt Size or even Time - helps compare the value and effort between tickets in a much clearer way. Fibonacci numbers of course help the Scrum Master and Product Owner establish Scrum Artifacts such as burn down charts.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: "Value" is ambiguous here.
Would argue that the "value" of the ticket should be outside the scope of estimating - that is for the Product Owner to determine.
But what I think what you mean is the quantifiable estimation of the ticket, taking into account time, complexity, unknowns, etc. So you could just take "value" out of this sentence and it would still read coherently.

Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: Also 'artifact' is the Amercian spelling - it's not wrong but if you want to be pedantically British, it's "artefact" :)

File: _posts/2016-08-26-why-estimate [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+Velocity is obviously the main bonus of using a technique like the Fibonacci Sequence. An established team should agree on the effort of a piece of work and therefore amount of work they can pull into their sprint. Over time, this establishes a consistent velocity which helps the Product Owner create longer term roadmaps for work and the team ensure they don’t overcommit to work even when under pressure to do so.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: Take out obviously. If it was obvious we would all be doing it.

File: _posts/2016-08-26-why-estimate [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Thinking outside the Scrum Team themselves, estimations helps manage stakeholder expectations. It gives a tool to the Product Owner to help them assess when a project or piece of work could be done and whether or not it’s viable.
+
+Estimation has many other pros - it helps show if a ticket is too large and needs to be broken down into smaller, more manageable parts; helps set sprints and makes it easier if there are scope changes later to swap similar effort work in for another; and it helps show quickly if a project or ticket is beyond the scope of the team and therefore needs more people or time dedicated to it.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: Yes yes yes - but is it the estimation itself, or the process of estimating (i.e. the discussion) that yields this insight?

File: _posts/2016-08-26-why-estimate [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+##...and in the blue corner, arguments against##
+
+There was a feeling that a lack of flexibility in deadlines, particularly when dealing with third party or stakeholder enforced deadlines, that this leads to forced velocity. There is no point in estimating if everything has to be done, regardless of whether it’s actually possible.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: The first sentence here doesn't flow well. Worth revisiting to make this clearer.

File: _posts/2016-08-26-why-estimate [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+Velocity only works as a measure with an established team, which is something that needs to be kept in mind if you have a team which constantly moves around, or changes the kind of work or platform they work on. There’s a variety in the way we estimate and with too many different methods it’s impossible to estimate effectively when joining a new team. What’s best? Do we need a Web Team “standard” to help when people transition teams? Context switching between methods of estimating when people move teams can be really difficult.
+
+If teams are overestimating, there’s a danger how downtime is used: some downtime is great for supporting colleagues, doing personal development or planning future projects, but overestimating to create extra downtime does not maximise productivity.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: And really calls into question the value of the estimation activity. Is the risk in over-estimating though, or in underestimating? (Remembering that optimism bias!)

File: _posts/2016-08-26-why-estimate [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+“Re-estimation wastes time and moves milestones” - there was a thought that re-estimating is just extra process and takes you away from the original goal you are working towards.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: You bring re-estimation in here as a negative, but there's no positive in this. So as a reader I would be questioning, well why do it in the first place?

File: _posts/2016-08-26-why-estimate [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+Other arguments we encountered were that estimating in time increases expectations from stakeholders as it’s often incorrect and more a ‘finger in the air’ estimate, and that the Fibonacci scale is essentially just time in some eyes.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: Break this sentence down, it's too long.

File: _posts/2016-08-26-why-estimate [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+Estimating is difficult when you have a variety of knowledge and experience in the team. Do you estimate it based on anyone picking up the ticket? If you have work that one dev can do faster, pressure is on Product Leads to allocate work to that dev to get it done quickly, which decreases learning opportunities.
+
+##Conclusion##
+Essentially, it boils down to this: we can’t give you a definitive answer! There are many pros to estimation, but many organisations and teams proactively get just as much work done without estimating their tickets. Estimation is not a required method of Scrum, but the fact so many people think it is shows how powerful a tool it is. After much discussion, we believe, much like the rest of Agile, that estimation as a methodology adapts around us and how we like to work so there isn't a one size fits all approach.
+
+One thing we all agree on is that if you’re going to estimate, it must be done blindly either with cards or hands or written down. Otherwise you get bias towards the first number stated and that means that we don’t discuss the reason for differing opinions fully and potentially miss some detail about the work.
```
Comment (carolinewoodhams) [inline _posts/2016-08-26-why-estimate]: "anchoring" = this bias

**Comment by carolinewoodhams** [comment]
Great blog post, good to share our learning around estimation with the tech community. The content is all there, just some grammar/readability issues to revise and then I think it would be ready to go. Let me know when you've made the changes and I will review again.

**Comment by hxmattobee** [comment]
I'll take the second spot

**Comment by hxmattobee** [comment]
The post needs a `.md` file extension.

File: _posts/2016-08-26-why-estimate [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+title: Why estimate
+categories: estimate
+author: nick_han
+tags: estimates
+comments: true
+excerpt: Is estimating ultimately a waste of time?
+---
+
+
+##What we did##
```
Comment (hxmattobee) [inline _posts/2016-08-26-why-estimate]: These headings need two hashes at the start only, not at the end:
`## What we did`

File: _posts/2016-08-26-why-estimate (hunk 1 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Done

File: _posts/2016-08-26-why-estimate (hunk 2 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Done

File: _posts/2016-08-26-why-estimate (hunk 3 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Removed 'completely' as this didn't add anything.

We've rephrased this sentence around new starters

File: _posts/2016-08-26-why-estimate (hunk 4 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Removed first reference of value to 'score'.

Scrum Artifacts is the official Scrum Alliance terminology, so we've kept as is.

File: _posts/2016-08-26-why-estimate (hunk 5 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Changed to 'usually'

File: _posts/2016-08-26-why-estimate (hunk 6 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Don't feel we can change this as it was other's opinions.

File: _posts/2016-08-26-why-estimate (hunk 7 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: We've rephrased this one.

File: _posts/2016-08-26-why-estimate (hunk 8 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Agreed, but again this is someone's opinion so leaving as is.

File: _posts/2016-08-26-why-estimate (hunk 9 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: We've deleted as this was not raised in the 'Pros' section and didn't add any value anyway.

File: _posts/2016-08-26-why-estimate (hunk 10 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Done.

File: _posts/2016-08-26-why-estimate (hunk 11 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: We think bias is more widely understood than anchoring, so think this best left as is thanks.

File: _posts/2016-08-26-why-estimate (hunk 12 above)
Comment (nickloestartup) [inline _posts/2016-08-26-why-estimate]: Done.

**Comment by nickloestartup** [comment]
Hi @carolinewoodhams and @hxmattobee.

Thanks for your comments. We've had a run through and made some changes if you could take another look and +1.

Matt, the "needs a .md file extension." comment, Jack will sort with us when we publish it, if we've not addressed that correctly already.

Thanks
Nick & Han

File: _posts/2016-08-26-why-estimate.md [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+
+If the discussion of the ticket is what’s important for the team, then the argument could be that we stop at the end of that discussion. However, we’ve found that putting a score on it - be that a Fibonacci, T-Shirt Size or even Time - helps compare the value and effort between tickets in a much clearer way. Fibonacci numbers of course help the Scrum Master and Product Owner establish Scrum Artifacts such as burn down charts.
+
+Velocity is usually the main bonus of using a technique like the Fibonacci Sequence. An established team should agree on the effort of a piece of work and therefore amount of work they can pull into their sprint. Over time, this establishes a consistent velocity which helps the Product Owner create longer term roadmaps for work and the team ensure they don’t overcommit to work even when under pressure to do so.
+
+Thinking outside the Scrum Team themselves, estimations helps manage stakeholder expectations. It gives a tool to the Product Owner to help them assess when a project or piece of work could be done and whether or not it’s viable.
```
Comment (hxmattobee) [inline _posts/2016-08-26-why-estimate.md]: Should "estimations helps manage" instead be "estimation helps manage.."?

Comment (HanCork89) [inline _posts/2016-08-26-why-estimate.md]: Yep, fixed!

**Comment by hxmattobee** [comment]
👍
